### DISCUSSION TITLE
Error In Bid Price

### DISCUSSION THREAD

**Discussion by krassensteinBrian** [deso-protocol/dips#116] [Medium CIPs]
There is a rather urgent glitch in the BitClout bidding system I noticed. If someone cancels their bid it still shows that the highest bid is the canceled bid. This is troublesome as someone can bid up an auction, cancel their bid and unless users click to view all the bids, it appears the auction price is higher than it actually is. This can prevent additional bidding.

See Below:

It says the highest bid is $59 even though it was $50.

https://bitclout.com/posts/f46c5327e7e408c51c86564373dc59d301660c9d9c103be3a489491b94f55735?tab=posts

**Comment by git-samhorine** [comment]
i wonder if this is the same issue I brought up yesterday in a post - though the conclusion that I found by testing was that it wasn’t about a user canceling their bid it was that a user made a NFT bid w/ funds available in wallet and at a later point didn’t have those same funds still available and so as result the bid was then hidden. 

Did you get a notification that bid was cancelled or is this a case of bidding w/ funds no longer available (which is also an issue)

**Reply by krassensteinBrian** [reply]
No!  I think you are right.  I just figured that was the case.
